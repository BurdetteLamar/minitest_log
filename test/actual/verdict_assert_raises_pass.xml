<log>
  <summary verdicts='2' failures='0' errors='0'/>
  <verdict method='verdict_assert_raises?' outcome='passed' id='0'>
    <error_class class='Class' value='RuntimeError'/>
  </verdict>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
