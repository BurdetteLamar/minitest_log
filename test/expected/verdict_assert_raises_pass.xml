<log>
  <summary verdicts='1' failures='0' errors='0'/>
  <verdict method='verdict_assert_raises?' outcome='passed' id='0'>
    <error_class class='Class' value='RuntimeError'/>
  </verdict>
</log>
